k8s/demo/common/probate/probate-back-office.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/probate/back-office:prod-22222a28
+      image: hmctspublic.azurecr.io/probate/back-office:latest
```

Comment:
I think this should be pr-1413-dddaec01 shouldn't it so that we don't get overwritten with master or some other image? Or is this ensured with the pr-1413-* tag in annotations?